src/Compression/CompressedReadBufferFromFile.cpp

```diff
@@ -91,6 +91,8 @@ void CompressedReadBufferFromFile::seek(size_t offset_in_compressed_file, size_t
 size_t CompressedReadBufferFromFile::readBig(char * to, size_t n)
 {
     size_t bytes_read = 0;
+    ICompressionCodec::CodecMode decompress_mode = ICompressionCodec::CodecMode::Synchronous;
```

Comment:
maybe add a comment at the end of the line that the codec mode is only relevant for codecs which support hardware offloading?

Reply:
Done